libraries/AP_ESC_Telem/AP_ESC_Telem.cpp

```diff
@@ -492,6 +492,18 @@ void AP_ESC_Telem::update_telem_data(const uint8_t esc_index, const AP_ESC_Telem
         telemdata.usage_s = new_data.usage_s;
     }
 
+#if AP_EXTENDED_ESC_TELEM_ENABLED
+    if (data_mask & AP_ESC_Telem_Backend::TelemetryType::INPUT_DUTY) {
+        _telem_data[esc_index].input_duty = new_data.input_duty;
```

Comment:
Should `_telem_data[esc_index]` be replaced with `telemdata`? When we opened the original PR you based this, we did not notice the changes made in #26252 

I can open the PR to fix that if you think we should.

Reply:
@robertlong13 yeah your right. A PR would be great, thanks!